site/_posts/2016-06-03-update-on-jekyll-s-google-summer-of-code-projects.markdown

```diff
@@ -0,0 +1,19 @@
+---
+layout: news_item
+title: "Jekyll's Google Summer of Code Project: The CMS You Always Wanted"
+date: "2016-06-03 13:21:02 -0700"
+author: parkr
+categories: [community]
+---
+
+This year, Jekyll applied to be a part of [Google Summer of Code](https://summerofcode.withgoogle.com/how-it-works/). Students were able to propose any project related to Jekyll. With a gracious sponsorship from GitHub and the participation of myself, [@benbalter](https://github.com/benbalter) and [@jldec](https://github.com/jldec), Jekyll was able to accept two students for the 2016 season, [@mertkahyaoglu](https://github.com/mertkahyaoglu) and [@rush-skills](https://github.com/rush-skills).
```

Comment:
Didn't we find that `jemoji` and `jekyll-mentions` mucked with the HTML document? :shrug: